### ISSUE TITLE
APC can't be deconstructed

### ISSUE THREAD

**Issue by Storm-Cloud** [yogstation13/yogstation#2193]
I followed the guide for deconstructing a damaged APC, smashed it until the cover came off (used a crowbar, wrench, welding tool, fire extinguisher, etc. ), took out the battery, loosened the electronics (screwdriver). Then I tried to cut the wires, and got "You have to uncover the power controller first", used the crowbar, "you have to remove the wires first".

When I used the wirecutters, it was acting like it was trying to do the second wirecutter step, instead of the first one.

**Comment by Time-Green** [comment]
That was the decon path for a broken APC, was the APC broken?

**Comment by Storm-Cloud** [comment]
yes it was broken, I don't know anything Admin-wise I was just playing, there weren't/aren't any admins on.

**Comment by Slinky5890** [comment]
I believe I also ran into this bug before and totally forgot about it. Ended up admin deleting it (with permission of course)

**Comment by Time-Green** [comment]
I actually had this bug with a derelict nutrivent...

**Comment by 00aquir** [comment]
did you crowbar the floor?
